src/Sylius/Bundle/AdminBundle/Resources/views/Currency/_form.html.twig

```diff
@@ -0,0 +1,10 @@
+{{ form_errors(form) }}
+<div class="one field">
+    <div class="required field">
+        <label class="required" for="sylius_currency_code"> Name </label>
```

Comment:
I also vote for not extending `AddCodeFormSubscirber`. We should do something like 
```twig
{{ form_label(form.code, sylius.ui.name|trans) }}
```